packages/miniflare/src/plugins/core/node-compat.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Wrangler can provide Node.js compatibility in a number of different modes:
+ * - "legacy" - this mode adds compile-time polyfills that are not well maintained and cannot work with workerd runtime builtins.
+ * - "als": this mode tells the workerd runtime to enable only the Async Local Storage builtin library (accessible via `node:async_hooks`).
+ * - "v1" - this mode tells the workerd runtime to enable some Node.js builtin libraries (accessible only via `node:...` imports) but no globals.
+ * - "v2" - this mode tells the workerd runtime to enable more Node.js builtin libraries (accessible both with and without the `node:` prefix)
+ *   and also some Node.js globals such as `Buffer`; it also turns on additional compile-time polyfills for those that are not provided by the runtime.
+ *  - null - no Node.js compatibility.
+ */
+export type NodeJSCompatMode = "legacy" | "als" | "v1" | "v2" | null;
+
+/**
+ * Computes the Node.js compatibility mode we are running.
+ *
+ * NOTE:
+ * Currently v2 mode is configured via `nodejs_compat_v2` compat flag.
+ * At a future compatibility date, the use of `nodejs_compat` flag will imply `nodejs_compat_v2`.
```

Comment:
I know the logic accounts for this, but we should update the comment re the "future" compatibility date

Reply:
Yes! I changed this in the Wrangler code but forgot this here.